README.md

```diff
@@ -20,13 +25,13 @@ Clone repository to: `$GOPATH/src/github.com/hashicorp/terraform-provider-$PROVI
 
 ```sh
 $ mkdir -p $GOPATH/src/github.com/hashicorp; cd $GOPATH/src/github.com/hashicorp
-$ git clone git@github.com:hashicorp/terraform-provider-$PROVIDER_NAME
+$ git clone git@github.com:hashicorp/terraform-provider-azure
```

Comment:
While you're here, might be good to change `hashicorp` to `terraform-providers`, and same below.